src/Translatable/Translatable.php

```diff
@@ -260,6 +261,40 @@ public function fill(array $attributes)
     }
 
     /**
+     * Get the fillable attributes for the model.
+     *
+     * @return array
+     */
+    public function getFillable()
+    {
+        $fillable = parent::getFillable();
+        $locales = $this->getLocales();
+
+        foreach ($this->translatedAttributes as $attribute) {
+            if ($this->alwaysFillable() || $this->isFillable($attribute)) {
+                foreach ($locales as $locale) {
+                    $fillable[] = $this->getFillableLocalizedAttributeKey($attribute, $locale);
+                }
+                unset($fillable[array_search($attribute, $fillable)]);
```

Comment:
If we are unsetting the attribute, why set it in the first place?

I don't like the fact we're looping through the locales here. A locale has nothing to do with the fact this field is fillable or not.

What we could do instead is change the syntax of the $fillable array for translated properties. Instead of doing this:

```php
public $fillable = ['code','name'];
```

We could do this:

```php
public $fillable = ['code','translation.name'];
```

What do you think?